### DISCUSSION TITLE
Add insert file next to image in WYSIWYG

### DISCUSSION THREAD

**Discussion by Kainkainkain** [directus/directus#4766] [Feature Requests]
Wanting to use Directus as a headless CMS. 

Simple use case: 
**User would like to, within text body, insert a PDF file and link to it. **

The task is simple in Wordpress. 

As a developer I have modified other CMS's to duplicate the insert image component and modified it to fit. 
The ultimate usability would be adding the insert file option next to insert image and altering the hyperlink button also with a prompt for a file link.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
